src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1475,9 +1475,11 @@ def list_vmss_instance_connection_info(resource_group_name, vm_scale_set_name):
     public_ip_address = public_ip.ip_address
 
     # loop around inboundnatrule
-    instance_addresses = []
+    instance_addresses = {}
```

Comment:
I know the table transformers have to use OrderedDict or the columns appear randomly. But yes, if you return an OrderedDict and the CLI converts it to a dict, then yeah, no point. :)